packages/dds/tree/src/simple-tree/toMapTree.ts

```diff
@@ -119,34 +121,30 @@ export function mapTreeFromNodeData(
 		return undefined;
 	}
 
-	const mapTree = nodeDataToMapTree(
-		data,
-		normalizedFieldSchema.allowedTypeSet,
-		schemaValidationPolicy,
-	);
+	const mapTree = nodeDataToMapTree(data, normalizedFieldSchema.allowedTypeSet);
+	// Add what defaults can be provided. If no `context` is providing, some defaults may still be missing.
 	addDefaultsToMapTree(mapTree, normalizedFieldSchema.allowedTypes, context);
+
+	// TODO:
+	// Since some defaults may still be missing, this can give false positives when context is undefined but schemaValidationPolicy is provided.
+	if (schemaValidationPolicy?.policy.validateSchema === true) {
+		inSchemaOrThrow(schemaValidationPolicy, mapTree);
+	}
+
 	return mapTree;
 }
 
+/**
+ * Copy content from `data` into a MapTree.
+ * Does NOT generate and default values for fields.
+ * Often throws UsageErrors for invalid data, but may miss some cases.
+ * @remarks
+ * Output is likely out of schema even for valid input due to missing defaults.
+ */
 function nodeDataToMapTree(
 	data: InsertableContent,
 	allowedTypes: ReadonlySet<TreeNodeSchema>,
-	schemaValidationPolicy: SchemaAndPolicy | undefined,
-): ExclusiveMapTree;
-function nodeDataToMapTree(
-	data: InsertableContent | undefined,
-	allowedTypes: ReadonlySet<TreeNodeSchema>,
-	schemaValidationPolicy: SchemaAndPolicy | undefined,
-): ExclusiveMapTree | undefined;
-function nodeDataToMapTree(
-	data: InsertableContent | undefined,
-	allowedTypes: ReadonlySet<TreeNodeSchema>,
-	schemaValidationPolicy: SchemaAndPolicy | undefined,
-): ExclusiveMapTree | undefined {
-	if (data === undefined) {
```

Comment:
This case is not checked against the schema, and it can't be since the type of field is not known. Fortunately this case isn't needed and has been removed, which also avoids needing overloads.

As these are internal facing APIs, if we did need both APIs, having two functions instead of overloads would be more type safe, and easier to modify in the future, but as the second case was unneeded, it was simply removed.